llama_stack/core/library_client.py

```diff
@@ -153,7 +153,11 @@ def __init__(
 
         self.loop = asyncio.new_event_loop()
 
-    def initialize(self):
+        # Automatically initialize the client
+        self._auto_initialize()
+
+    def _auto_initialize(self):
```

Comment:
why?

Reply:
so i have added this to replace the need to invoke `initialize()` explicitly, and just do it by default

plz let me know how to proceed

cc @nathan-weinberg